ls8/cpu.py

```diff
@@ -7,7 +7,9 @@ class CPU:
 
     def __init__(self):
         """Construct a new CPU."""
-        pass
+        self.memory = [0] * 8
```

Comment:
check the spec here. also suggest following the naming convention (memory -> ram) so that the `ram_read` `ram_write` method names are more descriptive